package.json

```diff
@@ -31,13 +31,13 @@
     "aria-query": "^0.7.1",
     "chai": "^4.1.0",
     "emoji-regex": "^7.0.0",
-    "eslint-plugin-jsx-a11y": "^6.2.1",
-    "eslint-plugin-vue": "^5.2.2",
+    "eslint-plugin-jsx-a11y": "^6.2.3",
+    "eslint-plugin-vue": "^7.0.0-alpha.1",
```

Comment:
How worried should I be about this being alpha 1?

Reply:
I need to upgrade the engine.node to point to >= 8, but otherwise the official site references 7.0.0-alpha.1 by default (see https://eslint.vuejs.org/rules/)